bindings/2.2074/GeometryDash.bro

```diff
@@ -4798,16 +4812,22 @@ class EnhancedGameObject : GameObject {
     int m_singleFrame;
     // property 592
     bool m_animationOffset;
-    PAD = android32 0xf, win 0xf, android64 0xf, mac 0xf;
-
+    bool m_animationTriggered;
+    int m_unkAnimationInt;
+    int m_maybeAnimationVariableXInt;
+    int m_maybeAnimationVariableYInt;
     // property 214
     bool m_animateOnlyWhenActive;
     // property 444
     bool m_isNoMultiActivate;
     // property 99
     bool m_isMultiActivate;
-    PAD = android32 0x4, win 0x4, android64 0x8, mac 0x8;
-}
+    bool m_activated;
+    bool m_activatedByPlayer1;
+    bool m_activatedByPlayer2;
+    bool m_hasUniqueCoin;
+    cocos2d::CCColor3B m_unkAnimationColor;
```

Comment:
this is supposed to be ccColor3B not uppercase

Reply:
Fixed